src/styles/_404.less

```
@@ -1,3 +1,5 @@
+@import './_variables.less';
```

```suggestion
@import (reference) './_variables.less';
```

`(reference)` ensures that this piece of declared import is not included to the final compiled css bundle unless it's necessary